bundles/org.openhab.binding.miio/README.md

```diff
@@ -465,6 +465,7 @@ Currently the miio binding supports more than 300 different models.
 | Mi Water Purifier v4         | miio:basic       | [yunmi.waterpurifier.v4](#yunmi-waterpurifier-v4) | Yes       | Experimental support. Please report back if all channels are functional. Preferably share the debug log of property refresh and command responses |
 | Smartmi Ventilation System   | miio:basic       | [zhimi.airfresh.va2](#zhimi-airfresh-va2) | Yes       |            |
 | Smartmi Fresh Air System (Heating) | miio:basic       | [zhimi.airfresh.va4](#zhimi-airfresh-va4) | Yes       |            |
+| Mi Fresh Air Ventilator C1-80 | miio:basic       | [zhimi.airfresh.ua1](#zhimi-airfresh-ua1) | Yes       | Identified manual actions for execution<br />`action{"did":"filter-reset-filter-life","siid":4,"aiid":1,"in":[1.0]}`<br />Please test and feedback if they are working to they can be linked to a channel. |
```

Comment:
```suggestion
| Mi Fresh Air Ventilator C1-80 | miio:basic       | [zhimi.airfresh.ua1](#zhimi-airfresh-ua1) | Yes       | Identified manual actions for execution<br />`action{"did":"filter-reset-filter-life","siid":4,"aiid":1,"in":[1.0]}`<br />Please test and feedback if they are working so they can be linked to a channel. |
```

Reply:
yes indeed.. fixed this and the similar ones